client/src/components/Modals/SubmitPitch/index.tsx

```diff
@@ -131,109 +145,136 @@ const SubmitPitchModal: FC<SubmitPitchModalProps> = ({
 
     if (isUndefined(field)) {
       return true;
-    } else if (field instanceof Set && isEmpty(field)) {
+    } else if ((field instanceof Set && isEmpty(field)) || field === '') {
       return true;
     }
 
     return false;
   };
 
   return (
-    <Modal
-      trigger={<Button className="default-btn" content="Submit a Pitch" />}
-      open={isOpen}
-      className="submit-modal"
-      onOpen={() => setIsOpen(true)}
-      onClose={() => setIsOpen(false)}
-      {...rest}
-    >
-      <Modal.Header>
-        <span style={{ paddingRight: '10px' }}>Submit a pitch</span>
-        <Popup
-          content={HelperMessage()}
-          trigger={<Icon size="small" name="info circle" />}
-          position="bottom left"
-          wide="very"
-          hoverable
-        />
-      </Modal.Header>
-      <Modal.Content>
-        <Form id="submit-pitch" onSubmit={submitPitch}>
-          <Form.Group widths="equal">
-            <Form.Input
-              required
-              error={isFieldError(title)}
-              label="Title"
-              value={title}
-              onChange={(e, { value }) => setTitle(value)}
-              size="small"
+    <div>
+      <Toaster></Toaster>
```

Comment:
This can be abstracted to the index.tsx file at the root of of the `client/src` directory